CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## 21.26.1
 
+* Dont show breadcrumb item with no url ([PR #1324](https://github.com/alphagov/govuk_publishing_components/pull/1324))
```

Comment:
This shouldn't be included under `## 21.26.1`, it should be under an `## Unreleased` heading.